CMakeLists.txt

```diff
@@ -338,11 +348,12 @@ add_definitions(-DBORINGSSL_IMPLEMENTATION)
 
 if(BUILD_SHARED_LIBS)
   add_definitions(-DBORINGSSL_SHARED_LIBRARY)
-  # Enable position-independent code globally. This is needed because
-  # some library targets are OBJECT libraries.
-  set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 endif()
 
+# Enable position-independent code globally. This is needed because
+# some library targets are OBJECT libraries.
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
We may need to add new option to disable `CMAKE_POSITION_INDEPENDENT_CODE` build for other aws-lc users who don't want enable `CMAKE_POSITION_INDEPENDENT_CODE`. If we do, the new option is relatively duplicate to `-DCMAKE_POSITION_INDEPENDENT_CODE`.

Can we use `target_compile_options` to enable `-fPIC` instead of adding new CMake option in aws-lc?

```
include(aws-lc)
target_include_directories(${PROJECT_NAME} PRIVATE|PUBLIC $<TARGET_PROPERTY:crypto,INTERFACE_INCLUDE_DIRECTORIES>)
target_link_libraries(${PROJECT_NAME} PRIVATE|PUBLIC crypto)
```

Reply:
Can we provide a new option `ENABLE_PIC`, which has default value `ON`?

I just discussed with my teammates. We think `PIC enable by default` is very common in today's situation.

Reply:
Updated https://github.com/awslabs/aws-lc/pull/113